Makefile

```diff
@@ -22,22 +25,40 @@ doccheck:
 	@cd Documentation; for doc in *.md; do if [ "$$doc" != "README.md" ] && ! grep -q "$$doc" *.md; then echo "ERROR: No link to documentation file $${doc} detected"; exit 1; fi; done
 	@echo OK
 
-deps:
-	go get github.com/tools/godep
+build: clean
+	GOOS=$(shell uname -s | tr A-Z a-z) GOARCH=$(ARCH) $(BUILDENVVAR) go build -o kube-state-metrics
+
+test-unit: clean build
+	GOOS=$(shell uname -s | tr A-Z a-z) GOARCH=$(ARCH) $(TESTENVVAR) go test --race $(FLAGS) $(PKGS)
+
+TEMP_DIR := $(shell mktemp -d)
+
+all: all-container
+
+sub-container-%:
+	$(MAKE) ARCH=$* container
```

Comment:
Prefix `$(MAKE)` with `@`, i.e., `@$(MAKE)`. This way make will not print something like `make[1]: Leaving directory '/home/tiger/gopath/src/k8s.io/kube-state-metrics'`

Reply:
Updated